custom_components/google_home/translations/de.json

```diff
@@ -6,7 +6,8 @@
         "description": "Hilfe zur Konfiguration findest du unter https://github.com/leikoilja/ha-google-home. Benutze deinen Google Account und ein App-Passwort. Es ist einfacher und sicherer, ein App-Passwort zu generieren und zu nutzen als das eigentliche Passwort. Es hat weiterhin die selben Zugriffsrechte, ist aber einfacher in der Handhabung beim Skripten als das echte Passwort. (https://myaccount.google.com/apppasswords). Falls nicht, dein normales Google Account Passwort sollte funktionieren.",
         "data": {
           "username": "Google Account Nutzername",
-          "password": "Google Account App-Passwort"
+          "password": "Google Account App-Passwort",
+          "master_token": "Google Account Master Token"
```

Comment:
is this a valid german translation? oh well, i guess "master token" is going to be the same in all languages 😅

Reply:
Also think it is going to be the same :D 
For german there is no good translation for "token", which is why we normally just use the word "token" in texts/speech